docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
Sure: the specific issue I see is the conflation of e.g. runic `:ifno` and emitted `[%if ...]`, and I'm not sure what a non-confusing way to present that would be. The old style(see e.g. the `$fry` expansion comment nearby) uses `{}` to represent quasi-unquote, invoking interpolation syntax, but due to recent damage to the grammar that now means `$:`.